### ISSUE TITLE
Change term - originalNameUsage & originalNameUsageID

### ISSUE THREAD

**Issue by mdoering** [tdwg/dwc#360]
## Term change

* Submitter:  Markus Döring
* Efficacy Justification (why is this change necessary?): Current definitions of originalNameUsage and originalNameUsageID contradict each other, although their core meaning should be the same.

Current Term definitions: 
https://dwc.tdwg.org/list/#dwc_originalNameUsage
https://dwc.tdwg.org/list/#dwc_originalNameUsageID

We need to discuss what the core meaning of an original name usage is and provide a clear definition, see discussion at https://github.com/tdwg/dwc/issues/105 and https://github.com/tdwg/dwc/issues/352

It could be either 
 - a basionym relation
 - a basionym relation or pointing to itself in case there is no basionym
 - a basionym, itself or a replacement name

**Comment by tucotuco** [comment]
This issue needs to have suggestions for the specific changes to be adopted.

**Comment by tucotuco** [comment]
This proposal is in danger of closure for lack of activity. If there is still interest in this? If so, we need concrete change requests to prepare it for public commentary.

**Comment by mdoering** [comment]
Thanks @tucotuco. I do still have interest in aligning the definitions of the 2 terms;)
Here is a proposal that only changes the `originalNameUsage` term, excluding replacement names from the originalNameUsage definition so it aligns with the existing originalNameUsageID term as it is:


## originalNameUsage

_Definition_ : The original taxon name, with authorship and date information if known, which first established the terminal element of the scientificName under the rules of the associated nomenclaturalCode.

_Comments_: The full scientific name, with authorship and date information if known, of the name usage in which the terminal element of the scientificName was originally established under the rules of the associated nomenclaturalCode. For example, for names governed by the ICNafp, this term would indicate the basionym of a record representing a subsequent combination. Unlike basionyms, however, this term can apply to scientific names at all ranks. It should not be used for replacement names to document the senior/earlier homonym.

_Examples_: `Pinus abies`, `Gasterosteus saltatrix Linnaeus 1768`, `Poa annua L.`

-----

For convenience here is the current definition of the **originalNameUsageID** term which should remain as it is:

_Definition_ : An identifier for the name usage (documented meaning of the name according to a source) in which the terminal element of the scientificName was originally established under the rules of the associated nomenclaturalCode.

_Comments_: This term should be used to refer to the taxonID of a Taxon record that represents the usage of the terminal element of the scientificName as originally established under the rules of the associated nomenclaturalCode. For example, for names governed by the ICNafp, this term would establish the relationship between a record representing a subsequent combination and the record for its corresponding basionym. Unlike basionyms, however, this term can apply to scientific names at all ranks. For Darwin Core Archives the related record should be present locally in the same archive.

_Examples_: `tsn:41107` (ITIS), `urn:lsid:ipni.org:names:320035-2` (IPNI), `2704179` (GBIF), `6W3C4` (COL)

**Comment by deepreef** [comment]
I'm certainly in favor of aligning the definitions of the two terms.  I'm generally in favor of the proposal @mdoering suggest for modifying `originalNameUsage` to match the meaning of `originalNameUsageID`.  However, I'm a little confused by the misalignment with respect to replacement names.  Why is this specified in the proposed comments for `originalNameUsage`, but not `originalNameUsageID`?  Also, what is the rationale for excluding replacement names from the scope of `originalNameUsage`?

Also note: this definition (for both terms) is similar to, but different from "Protonym", in that "Protonym" applies to the first chronological usage (not necessarily the first Code-compliant usage), and also regards replacement names as distinct from the names they replace.

**Comment by mdoering** [comment]
> However, I'm a little confused by the misalignment with respect to replacement names. Why is this specified in the proposed comments for `originalNameUsage`, but not `originalNameUsageID`?

Do you mean the proposed comment "It should not be used for replacement names to document the senior/earlier homonym."? I thought it is simpler to leave the originalNameUsageID term as it is, thats why such a comment is not present there. My main worry is to make clear the terms do not get used for replacement names which the current definition for originalNameUsage explicitly includes.

The requirement of code compliancy makes sense to me, but might be worth discussing further. I would like to avoid different use of this term which makes aggregation of data hard to impossible. Working with datasets which used dwc:originalNameUsage to refer to replaced names was what mostly triggered me to create this issue. [Basionyms in the botanical code](https://www.iapt-taxon.org/nomen/pages/main/glossary.html) are required to be legitimate: 
> **basionym**. The legitimate, previously published name on which a new combination or name at new rank is based. The basionym does not itself have a basionym; it provides the final epithet, name, or stem of the new combination or name at new rank ([Art. 6.10](https://www.iapt-taxon.org/nomen/pages/main/pages/main/art_6.html#Art6.10)) (see also [name at new rank](https://www.iapt-taxon.org/nomen/pages/main/pages/main/glossary.html#name%20at%20new%20rank), [new combination](https://www.iapt-taxon.org/nomen/pages/main/pages/main/glossary.html#new%20combination)).

Finding earlier, invalid names can be done from relations of the basionym. Similar to finding replaced names from the replacement name. It seems less confusing to stick with the basionym and current term definitions. But maybe there are arguments for the inclusion of invalidly published names @deepreef ? I am a bit scared without the code in place to help this increases uncertainty what the original name actually is.

**Comment by deepreef** [comment]
>  I thought it is simpler to leave the originalNameUsageID term as it is, thats why such a comment is not present there. 

Ah!  That makes sense,  But in order to keep them in alginment, I supposed one would also need to propose to amend the Comment of `originalNameUsageID` (if alignment is the goal).  Although... it would seem to me that the inclusion or exclusion of replacement names might rise to the level of a "definitional" thing, rather than a qualifier in the comments?

As for whether to include or exclude replacement names, I guess I'd need to understand the botanical Code better. Consider this example:

The name Aus bus Smith is proposed for a new species of plant.
The same name Aus bus Jones is later proposed for an entirely different species of plant.

Aus bus Jones is a homonym of Aus bus Smith, so Aus cus Brown is proposed as a replacement for Aus bus Jones.

Later, Aus cus Brown is placed in the genus Xus by Pyle, resulding in Xus cus (Brown) Pyle.

Is that how it works in Botany with replacement names (in terms of authorship allocation)?

Or, is Aus bus Jones considered the basionym of Aus cus Brown? [in which case I assume it would be Aus cus (Jones) Brown and  Xus cus (Jones) Pyle]

Are you saying you would like the definition of `originalNameUsage`/`originalNameUsageID` to essentially align with the definition of "basionym", except extended to higher ranks?  If so, then what about the fact that a basionym does not itself have a basionym -- does that mean that an `originalNameUsage` does not itself have an `originalNameUsage`?

Just to be clear, I am not opposed to defining `originalNameUsage`/`originalNameUsageID` in the way you propose (i.e. first Code-compliant usage, extending to higher ranks but not including replacement names).  I just want to make sure I understand the logic and rationale behind defining those terms in that way.

Perhaps it would help to have specific examples?

**Comment by mdoering** [comment]
> Later, Aus cus Brown is placed in the genus Xus by Pyle, resulding in Xus cus (Brown) Pyle.

If I understand [article 58 of the code](https://www.iapt-taxon.org/nomen/pages/main/art_58.html) correctly that is actually not the case. It would simply be _Xus cus_ Pyle and priority for cus starts with the Pyle publication.

Not also that in botany you are required to cite the basionym or replaced synonym to validly publish a new combination or replacement name.

**Comment by mdoering** [comment]
> Are you saying you would like the definition of `originalNameUsage`/`originalNameUsageID` to essentially align with the definition of "basionym", except extended to higher ranks? If so, then what about the fact that a basionym does not itself have a basionym -- does that mean that an `originalNameUsage` does not itself have an `originalNameUsage`?

I guess that is true indeed. My understanding of the exiting definition at least was always close to basionym. The definition says it needs to be originally _established_ under the associated code. That is a bit open to interpretation I suppose. Is an established name in botany a validly published one or a legitimate one?

> An identifier for the name usage in which the terminal element of the scientificName was originally established under the rules of the associated nomenclaturalCode.

Whether we can use originalNameUsage/ID to also refer to itself (and then strictly not be a basionym) I don't care much about. These cases are very easily detected. Inclusion/exclusion of replacement names is important to know though. If the end user wants to keep replacement names apart from "basionyms" (like the botanical code does) that will then not be easy.

My understanding of protonym also was that it does not include replaced names.

**Comment by deepreef** [comment]
> If I understand [article 58 of the code](https://www.iapt-taxon.org/nomen/pages/main/art_58.html) correctly that is actually not the case. It would simply be Xus cus Pyle and priority for cus starts with the Pyle publication.

Wait... really?  Why would Pyle get credit for the species epithet cus, when it was legitimately first established by Brown?  All pyle did was take the existing cus, originally established by Brown within the genus Aus, and created the new combination of Brown's cus within the genus Xus.

Note: I'm assuming that Pyle would have cited Brown when creating the new combination.

> That is a bit open to interpretation I suppose. Is an established name in botany a validly published one or a legitimate one?

Good point!  I've always used the term "established" as meaning "in accordance with the rules of the relevant Code".  I use the term "proposed" when it might have been Code compliant or it might not have been. (thus, all established names were proposed, but not all propsoed names were established).  I don't off-hand know the difference between "validly published" and "legitimate" in the ICNafp context.  We use "available" in zoology to mean "established in compliance with all Code rules".  I always thought that ws equivalent to "validly published", but I don't know what the differennce with "legitimate" is.

> My understanding of protonym also was that it does not include replaced names.

Protonyms definitely include replacement names.  A Protonym (sensu TNUs/etc.) is considered the first chroological proposal of a scientific name, regardless of whether it is Code compliant.  The only functional difference with replacement names is that they objectvely inherit the name-bearing type of the replaced name.  But they are still proposed in TNUs as "new names", and as such these proposals for replacement names are Protonyms.

It can be a bit confusing because replacement names are (senus stricto) "homotypic synonyms" because they share the same name-bearing type.  However, that term "homotypic synonym" is often used to mean "alternate combinations of the same species within different genera".  In the case of replacement names, following the Protonym model, they are also homotypic synonyms, except do not share the "same" terminal epithet (in this case, "same" refers to Protonym instance, rather than literal spelling).

This is something Dave Remsen, Nicolas Bailly and I worked through in some detail relating to our taxonomy modelling.

As a final note: there are other cases of homotypic synonyms that are not alternate combinations (i.e., involve multiple protonyms).  In rare cases, two different authors propose different new species names based on the same name-bearing type specimens.  The same also happens when two different authors assign the same species as the type for two different new genera. In other examples, sometimes authors will deliberately designate a name-bearing type of one name to be the neotype of another name.

So... technically speaking:  Homotypic synonyms include alternate combinations of the same terminal name, replacement names, inadvertent or intentional assignment of the same name-bearing type to two different names, and specific examples of neotype designations.  Collectively, we refer to these as "objective synonyms" in zoology -- not sure off-hand what term is used (if any) under ICNafp.

**Comment by mdoering** [comment]
> > If I understand [article 58 of the code](https://www.iapt-taxon.org/nomen/pages/main/art_58.html) correctly that is actually not the case. It would simply be Xus cus Pyle and priority for cus starts with the Pyle publication.
> 
> Wait... really? Why would Pyle get credit for the species epithet cus, when it was legitimately first established by Brown? All pyle did was take the existing cus, originally established by Brown within the genus Aus, and created the new combination of Brown's cus within the genus Xus.
> 
> Note: I'm assuming that Pyle would have cited Brown when creating the new combination.

Sorry, I misread your example, skipped one name and thought Brown was the illegitimate name. Recapturing the publication events:

1. _Aus bus_ Smith from 1899
2. _Aus bus_ Jones  nom. illeg., a later homonym from 1904 (but validly published)
3. _Aus cus_ Brown 1905 published as a replacement for Aus bus Jones (cited in the publication)
4. _Xus cus_ (Brown) Pyle in 2022

Your authorships are as I would expect them, _Xus cus_ (Brown) Pyle included!


I found a proposal by Werner Greuther to better define replacement names in botany with some examples and good explanations: https://onlinelibrary.wiley.com/doi/pdf/10.12705/654.29

He states a replacement name behaves just like a new taxon in terms of authorship and priority, so Brown is cited as the basionym author and I would therefore also call  _Aus cus_ Brown the basionym of  _Xus cus_ (Brown) Pyle:

> Whether a name has been published as a replacement name or as a name of a new taxon is irrelevant for its form, authorship and date for purposes of priority. The single reason why it is useful to make a distinction is typification: a replacement name has the same type as its replaced synonym, a name of a new taxon has a type used or cited by its author.

**Comment by mdoering** [comment]
Slightly different [examples from the code](https://www.iapt-taxon.org/nomen/pages/main/art_58.html) which I found interesting, as the replacement name here maintains the epithet as it is being placed in a new genus to avoid the collision instead of coining a new epithet: 

> Ex. 1. The name Talinum polyandrum Hook. (in Bot. Mag.: ad t. 4833. 1855) is illegitimate under [Art. 53.1](https://www.iapt-taxon.org/nomen/pages/main/pages/main/art_53.html#Art53.1) because it is a later homonym of T. polyandrum Ruiz & Pav. (Fl. Peruv. Prodr.: 65. 1794). When Bentham (Fl. Austral. 1: 172. 1863) transferred T. polyandrum Hook. to Calandrinia Kunth, he called it C. polyandra. This name has priority from 1863, and is cited as C. polyandra Benth., not C. polyandra “(Hook.) Benth.”

> Ex. 2. Cymbella subalpina Hust. (in Int. Rev. Gesamten Hydrobiol. Hydrogr. 42: 98. 1942) is illegitimate under [Art. 53.1](https://www.iapt-taxon.org/nomen/pages/main/pages/main/art_53.html#Art53.1) because it is a later homonym of C. subalpina F. Meister (Kieselalg. Schweiz: 182, 236. 1912). When Mann (in Round & al., Diatoms: 667. 1990) transferred C. subalpina Hust. to Encyonema Kütz., he called it E. subalpinum D. G. Mann. This name is a replacement name with priority from 1990 and as such is illegitimate under [Art. 52.1](https://www.iapt-taxon.org/nomen/pages/main/pages/main/art_52.html#Art52.1) because C. mendosa VanLand. (Cat. Fossil Recent Gen. Sp. Diatoms Syn. 3: 1211, 1236. 1969) had already been published as a replacement name for C. subalpina Hust.

**Comment by ghwhitbread** [comment]
@mdoering, I wonder if your definition change for originalNameUsage really aligns it with originalNameUsageId.   It remains a name, while the usage identifier references a name accordingTo some source.  The general pattern in DarwinCore is to align *UsageID terms with accordingTo strings (name sensu source) while aligning *Usage terms with scientificName.  Maybe not the original intention, but If we want to align terms and maintain the established pattern, we may need another term for originalNameAccordingTo.

With originalName* terms restricted to the basionym (or basionym-like relationships in any code), we don’t have a way of referencing the usage (the accordingTo) establishing names that are not changed combinations or for changed botanical combinations - other than by inference through a scientificName and namePublishedIn.

@deepreef, Markus has shown how your example ends with the name  Xus cus (Brown) Pyle, but that is not the only use case for originalNameUsageID.  If Jones had legitimately published Zus bus Jones and Pyle moved it to Aus it would get a replacement name, Aus cus Pyle, because the epithet is occupied by Aus bus Smith.  If you then move it to Xus it will become Xus bus (Jones) Pyle. In both cases, the epithet is provided by the earliest legitimate name of the taxon at the same rank.

**Comment by ghwhitbread** [comment]
With respect to replacement names (nomen novum), the existing definition for originalNameUsage states “… or the senior/earlier homonym for replaced names.”   The existence of a senior homonym is the reason for the replacement name, so the originalNameUsage is the junior homonym - the replaced name.

**Comment by mdoering** [comment]
> With respect to replacement names (nomen novum), the existing definition for originalNameUsage states “… or the senior/earlier homonym for replaced names.” The existence of a senior homonym is the reason for the replacement name, so the originalNameUsage is the junior homonym - the replaced name, no?

Yes it does. But for the sister term originalNameUsageID it does not state that and from reading the definitions I would expect replacement names to be excluded:

> An identifier for the name usage (documented meaning of the name according to a source) in which the terminal element of the scientificName was originally established under the rules of the associated nomenclaturalCode.

> Comments | This term should be used to refer to the taxonID of a Taxon record that represents the usage of the terminal element of the scientificName as originally established under the rules of the associated nomenclaturalCode. For example, for names governed by the ICNafp, this term would establish the relationship between a record representing a subsequent combination and the record for its corresponding basionym. Unlike basionyms, however, this term can apply to scientific names at all ranks. For Darwin Core Archives the related record should be present locally in the same archive.

This contradiction between the 2 sister terms is my primary reason to fix the definitions so they do line up.

My choice would be to exclude replacement names, but we can obviously also go the other route an include them as long as its defined in both.

**Comment by mdoering** [comment]
> @mdoering, I wonder if your definition change for originalNameUsage really aligns it with originalNameUsageId. It remains a name, while the usage identifier references a name accordingTo some source. The general pattern in DarwinCore is to align *UsageID terms with accordingTo strings (name sensu source) while aligning *Usage terms with scientificName. Maybe not the original intention, but If we want to align terms and maintain the established pattern, we may need another term for originalNameAccordingTo.

My understanding is that *UsageID terms must refer to a local dwc:Taxon (=NameUsage) record identified by dwc:taxonID in the dataset. At least for dwc archives. *Usage terms also point to a dwc:Taxon, but by using a scientific name instead of an identifier. They are not called *Name, but *NameUsage. Instead of citing the ID you cite the "label". For dwc archives at least the big difference here is that there is no requirement that such a record must exist in the local dataset - but it can. To me the core meaning of all *Usage and *UsageID terms should always be the same, just the way to express the relation is different.  

When it comes to accordingTo it might make sense to accompany each of the name label *Usage terms  with an accordingTo term. Although the dataset itself with its metadata already provides a default accordingTo scope. It really is only needed to point to other sources. 

> With originalName* terms restricted to the basionym (or basionym-like relationships in any code), we don’t have a way of referencing the usage (the accordingTo) establishing names that are not changed combinations or for changed botanical combinations - other than by inference through a scientificName and namePublishedIn.
> 
> @deepreef, Markus has shown how your example ends with the name Xus cus (Brown) Pyle, but that is not the only use case for originalNameUsageID. If Jones had legitimately published Zus bus Jones and Pyle moved it to Aus it would get a replacement name, Aus cus Pyle, because the epithet is occupied by Aus bus Smith. If you then move it to Xus it will become Xus bus (Jones) Pyle. In both cases, the epithet is provided by the earliest legitimate name of the taxon at the same rank.

**Comment by mdoering** [comment]
I think it's worthwhile capturing the 2 Aus bus examples in examples how one would express these with DwC.

My example given using *Usage terms:
```
scientificName,namePublishedInYear,nomenclaturalStatus,acceptedNameUsage,originalNameUsage
Aus bus Smith,1899,sp.nov,,
Aus bus Jones,1904,nom. illeg.,Aus cus Brown,
Aus cus Brown,1905,nom.nov.,,
Xus cus (Brown) Pyle,2022,comb.nov,,Aus cus Brown
```

and the same using *UsageID terms:
```
taxonID,scientificName,namePublishedInYear,nomenclaturalStatus,acceptedNameUsageID,originalNameUsageID
1,Aus bus Smith,1899,sp.nov,,
2,Aus bus Jones,1904,nom. illeg.,3,
3,Aus cus Brown,1905,nom.nov.,,
4,Xus cus (Brown) Pyle,2022,comb.nov,,3
```

**Comment by mdoering** [comment]
@ghwhitbread's example in 2 flavours:

```
scientificName,namePublishedInYear,nomenclaturalStatus,originalNameUsage
Aus bus Smith,1899,sp.nov,
Zus bus Jones,1904,sp.nov,
Aus cus Pyle,1998,comb.nov,
Xus bus (Jones) Pyle,2022,nom.nov.,Zus bus Jones
```

```
taxonID,scientificName,namePublishedInYear,nomenclaturalStatus,originalNameUsageID
1,Aus bus Smith,1899,sp.nov,
2,Zus bus Jones,1904,sp.nov,
3,Aus cus Pyle,1998,comb.nov,
4,Xus bus (Jones) Pyle,2022,nom.nov.,2
```

**Comment by deepreef** [comment]
@ghwhitbread:
> If Jones had legitimately published Zus bus Jones and Pyle moved it to Aus it would get a replacement name, Aus cus Pyle, because the epithet is occupied by Aus bus Smith. If you then move it to Xus it will become Xus bus (Jones) Pyle. In both cases, the epithet is provided by the earliest legitimate name of the taxon at the same rank.

Yes, that makes sense.  I knew there were other pathways to homonymy, but I chose a very simple one for my example.  The other examply you give (Zus bus Jones later moved to Aus, preoccupied by Aus bus Smith) is what we in Zoology-land refer to as a "secondary homonym" (species epithet described legitimately, but later moved to a genus that creates a homonym "collision").  The example I gave is what we call a "primary homonym" (i.e., a name born as a homonym, with original genus and species combination already preoccupied).

@mdoering:
> He states a replacement name behaves just like a new taxon in terms of authorship and priority, so Brown is cited as the basionym author and I would therefore also call Aus cus Brown the basionym of Xus cus (Brown) Pyle

OK, good -- that makes sense!

> This contradiction between the 2 sister terms is my primary reason to fix the definitions so they do line up.
> 
> My choice would be to exclude replacement names, but we can obviously also go the other route and include them as long as its defined in both. See the 3 options listed in the top issue description.

So, I think there are several different things that should be incorporated into the definitions & comments of both terms:

1) Both terms should be in harmony with each other; whatever the definition is. `I assume all agree to this.`
2) Should the term only apply to species (and lower) ranks, or all ranks? `I think we agree on all ranks.`
3) Should it be understood to refer to a specific *usage* instance (TNU), rather than a "name" instance? `I **emphatically** support usage instance; see below.`
4) Should it apply to the first Code-compliant usage, or the first chronological usage? `It seems @mdoering would prefer Code-compliant usage, which is consistent with basionym and consistent with the way both terms have already been defined; distinguishes from Protonym, which is the first chronological usage.`
5) Should it apply only to new names for new taxa, or also include usages representing proposals for replacement names? `@mdoering indicates he prefers excluding replacement names; I *slightly* prefer on including them, but don't have a strong opinion.`
6) Should a row representing an `originalNameUsage` reference itself? `[Informatically, I think "yes" is more useful because it's explicit, and gives an easy way to select a list of originalNameUsage instances be filtering on taxonID = originalNameUsageID.]`

> Leaving accordingTo aside I would much appreciate ways to provide more name relations like replacement names and relations to invalid, earlier names. Either through a dedicated extension like the one we [drafted for COL](https://rs.gbif.org/sandbox/extension/col-name-relation.xml) or by creating new dwc terms like replacedNameUsage(ID) or basedOnNameUsage(ID). But that might step too much into TCS2 grounds already.

If we're ever going to make any progress on standardizing the way we exchange of taxonomic & nomenclatural information, we absolutely must remove the ambiguity of whether a term refers to a "Name" or a "Usage".  The definition of `dwc:taxonID` was intentionally (and at the time, wisely) left ambiguous because there was so much heterogeny in how people represented taxonomic information, but we've been stuck for a few decades largely because we can't seem to get past this heterogeny.

The various "usage" terms were established in DwC *explicitly* to accommodate a future world where taxonominc data providers would converge on usage instances (rather than "Name" instances) as the primary units of information exchange. Although that hasn't happened yet, I think we're getting mych closer to that world, with TCS2 and various other developments in taxonomic informatic landscape.

Honestly, I think we should fully abandon any efforts to assign identifiers to "names" as such.  We can think of Protonyms and/or `originalNameUsage` instances as proxies for "names", and we certainly want to capture "name" information in various properties of usage instances.  Keep in mind that `accordingTo` can be "unknown" for a usage instance, and this can be treated as missing data and later amended when the values are known.  It's better to have such a "placeholder" TNU instance than to pretned that we will ever achive consensus in the biodiversity community on how "names" are trackable objects on their own. 
In any case, and in this context, there are a number of ways the relationship between a replacement name and the name it replaces can be represented.

I've combined your second set of examples, representing proper usages and also using self-referential values.

```
taxonID,scientificName,namePublishedInYear,nomenclaturalStatus,originalNameUsageID,originalNameUsage
1,Aus bus Smith,1899,sp.nov,1,Aus bus Smith,1899 sec. Smith,1899
2,Zus bus Jones,1904,sp.nov,2,Zus bus Jones,1904 sec. Jones,1904
3,Aus cus Pyle,1998,nom.nov,3,Aus cus Pyle,1998 sec. Pyle,1998
4,Xus bus (Jones) Pyle,2022,comb.nov.,2,Zus bus Jones sec. Jones,1904
```
Note this also implies that the taxonID values represent usages, so each one of these would also have an `accordingTo` value referring to the "sec." authorship of the `originalNameUsage`, escept for 4 which would be `accordingTo` Pyle,2022.

**Comment by mdoering** [comment]
I agree with the list of decisions to take and also do not care much about number 6. Filtering out self referring records is trivial so it does not do any harm. To me the key decisions to take are therefore 4 (code compliancy) & 5 (replacement names). I had hoped to not cause too much noise by keeping one complete term definition as it is and only slightly refine the other to align. But if we can get an agreement on a better definition for both I am happy to do that. I would just like to avoid repeating lengthy TCS2 discussions here which could be endless. Changing definitions of dwc:Taxon is not in scope of this issue in my mind and should be better adressed in TCS2. Personally I have also never encountered problems with the ambiguity of dwc:Taxon and dwc:taxonID in real datasets. 

On the other hand the practice of including replacement names has caused serious problems for our systems. Mainly because I would expect a single (hmomotypic) originalName that all subsequent combinations directly refer to. But with replacement names I found "chained" relations where one originalName (the replacement name) further refers to another originalName (the replaced synonym/homonym).

**Comment by deepreef** [comment]
Agreed!  I'm also OK with your preferences for 4 & 5 (and agree that 6 is probably best to mention in the comments as a "best practices" thing).  But I think it's unavoidable that both terms need to be tweaked - at least at the comments (non-normative) level.  If it's possible to maintain one definition intact, and adjust the other definition to match, then that would be only one normative change.  Then the comments of both can be brought into alignment as needed.

I also agree that altering `dwc:taxonID` is out of scope (and unwise at this stage, for various reasons).

I would say on #4, I favor limiting it to Code-compliant "original" usages, as this gives us a useful term that is different from protonym.  That also seems to be harmonious with existing definitions, and common use.

On #5 (replacement names), I defer to you and @ghwhitbread and anyone else who has a strong opinion.  But I think there is some need to clarify the distinction between "validly published" and "legitimate", and which term best applies to `originalNameUsage`/`ID`.

**Comment by nielsklazenga** [comment]
I think the current definition is fine (-ish). The whole purpose of the term is to link homotypic synonyms. I would leave the second sentence out, but if it is left in, something like '..., or the dwc:scientificName itself', should be added.

**Comment by tucotuco** [comment]
This issue is still active, but unresolved. It lacks a (so-far) consensus [Term change request](https://github.com/tdwg/dwc/issues/new?assignees=&labels=Term+-+change&projects=&template=term-change-template.md&title=Change+term+-+) based on the discussion so far. Can the interested parties please provide that?
